src/styles/blocks/content.scss

```diff
@@ -0,0 +1,157 @@
+.star {
+  display: flex;
+}
+
+.stars_star {
+  display: flex;
+  margin-left: 4px;
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  background-position: center;
+}
+
+.stars--5 .stars_star:not(:nth-child(n + 6)),
+.stars--3 .stars_star:not(:nth-child(n + 4)),
+.stars--2 .stars_star:not(:nth-child(n + 3)),
+.stars--1 .stars_star:nth-child(1),
+.stars--4 .stars_star:not(:nth-child(n + 5)) {
+  background-image: url(../images/star-active.svg);
+}
+
+.star .stars_star:first-child {
+  margin-left: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  &__logo {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 40px;
+  }
+
+  &__title {
+    color: $black_text_color;
+    width: 166px;
+    height: 36px;
+    font-weight: 500;
+    font-size: 12px;
+    font-style: normal;
+    line-height: 18px;
+    margin: 0 16px 4px;
+  }
+
+  &__code {
+    width: 92px;
+    height: 14px;
+    color: #616070;
+    font-size: 10px;
+    font-weight: 400;
+    font-style: normal;
+    margin: 0 90px 16px 16px;
+  }
+
+  &__rate {
+    display: flex;
+    height: 16px;
+    margin: 0 16px 24px;
+  }
+
+  .feedback {
```

Comment:
Check the 8th point from the [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)